executor/point_get.go

```diff
@@ -332,7 +338,30 @@ func (e *PointGetExecutor) get(ctx context.Context, key kv.Key) ([]byte, error)
 		}
 		// fallthrough to snapshot get.
 	}
-	return e.snapshot.Get(ctx, key)
+
+	cacheDB := e.ctx.GetStore().GetMemCache()
+	isLocked := e.tblInfo.IsLocked()
+	if !isLocked || e.tblInfo.Lock.Tp != model.TableLockRead { // if the lock is read or tale was unlock then snapshot get
+		if cacheDB.IsExists(e.tblInfo.ID) {
+			cacheDB.Release(e.tblInfo.ID)
```

Comment:
I think it should be released when unlocking tables. Think about this,
lock tables
point select... -> data cached
unlock tables
update table
lock tables
point select... (will this still get the cached data?)

Reply:
you are right.

Reply:
addressed, release cache in ddl_api. PTAL